pkgs/applications/editors/emacs/elisp-packages/update-melpa.el

```diff
@@ -257,10 +257,8 @@ return Promise to resolve in that process."
       url (lambda (status)
             (funcall resolve (condition-case err
                                  (progn
-                                   (goto-char (point-min))
-                                   (search-forward "\n\n")
-                                   (message (buffer-substring (point-min) (point)))
-                                   (delete-region (point-min) (point))
+                                   (message "%S" (url-http-parse-headers))
```

Comment:
`(url-http-parse-headers)` returns `t` or `nil`.  Logging only `t` or `nil` seems a bit meaningless to me.

Reply:
You are right. The header contents should be logged instead. I pushed a commit for it.